src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt

```diff
@@ -182,7 +182,7 @@ class AdminController(
 
   @GetMapping("/plantingSite/{plantingSiteId}")
   fun getPlantingSite(@PathVariable plantingSiteId: PlantingSiteId, model: Model): String {
-    val plantingSite = plantingSiteStore.fetchSiteById(plantingSiteId)
+    val plantingSite = plantingSiteStore.fetchSiteById(plantingSiteId, true)
```

Comment:
`false`? Or is this the API to fetch all monitoring plots?

Reply:
This is for the admin UI, where we want to have the full data set (and where it doesn't matter if that ends up being huge).